spec/components/comments/votes_component_spec.rb

```diff
@@ -0,0 +1,65 @@
+require "rails_helper"
+
+describe Comments::VotesComponent do
+  let(:user) { create(:user) }
+  let(:comment) { create(:comment, user: user) }
+  let(:component) { Comments::VotesComponent.new(comment) }
+
+  describe "aria-pressed attribute" do
```

Comment:
In case the users are not logged in, I think the following test can be useful:
https://github.com/consuldemocracy/consuldemocracy/blob/master/spec/system/votes_spec.rb#L283

In the case of not having permissions because it is an organisation, do you think that the test that has been added in `spec/models/abilities/organisation_spec.rb:26` could be useful?